doc/faq/howto_faq.rst

```diff
@@ -527,58 +527,6 @@ You may be able to work on separate figures from separate threads.  However,
 you must in that case use a *non-interactive backend* (typically Agg), because
 most GUI backends *require* being run from the main thread as well.
 
-.. _howto-contribute:
-
-How-to: Contributing
-====================
-
-.. _how-to-contribute-docs:
-
-Contribute to Matplotlib documentation
---------------------------------------
-
-Matplotlib is a big library, which is used in many ways, and the
-documentation has only scratched the surface of everything it can
-do.  So far, the place most people have learned all these features are
-through studying the :ref:`examples-index`, which is a
-recommended and great way to learn, but it would be nice to have more
-official narrative documentation guiding people through all the dark
-corners.  This is where you come in.
-
-There is a good chance you know more about Matplotlib usage in some
-areas, the stuff you do every day, than many of the core developers
-who wrote most of the documentation.  Just pulled your hair out
-compiling Matplotlib for Windows?  Write a FAQ or a section for the
-:ref:`installing-faq` page.  Are you a digital signal processing wizard?
-Write a tutorial on the signal analysis plotting functions like
-:func:`~matplotlib.pyplot.xcorr`, :func:`~matplotlib.pyplot.psd` and
-:func:`~matplotlib.pyplot.specgram`.  Do you use Matplotlib with
-`django <https://www.djangoproject.com/>`_ or other popular web
-application servers?  Write a FAQ or tutorial and we'll find a place
-for it in the :ref:`users-guide-index`.  And so on...  I think you get the
-idea.
-
-Matplotlib is documented using the `sphinx
```

Comment:
Is this mentioned anywhere else?

Reply:
hmm, I think most of this language would be great for a blog post, go back and forth about whether it should be shifted to the contributing docs section of contributing.

Reply:
@QuLogic With "this" do you mean sphinx? If so, that is mentioned in https://matplotlib.org/devdocs/devel/documenting_mpl.html

Reply:
@story645 Are you referring to the complete section? IMHO this is too chatty for the docs.

Reply:
Yes, I meant about Sphinx, mostly. It could be a tiny bit more explicit, but yea I see it is in that page.